packages/p2p/src/components/orders/popup.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { Component }                   from 'react';
+import PropTypes                              from 'prop-types';
+import { Formik, Field, Form }                from 'formik';
+import { Checkbox, Button, ThemedScrollbars } from 'deriv-components';
+import { localize }                           from 'deriv-translations';
+import IconClose                              from '../../assets/icon-close.jsx';
+
+class Popup extends Component {
+    handleSubmit = (values, { setSubmitting }) => {
+        console.log(values)
+        this.props.onClickConfirm();
+        setSubmitting(false);
+    }
+
+    render() {
+        const {
+            onCancel,
+            onClickConfirm,
+            title,
+            need_confirmation,
+            offer,
+            message,
+            has_cancel,
+            cancel_text,
+            confirm_text,
+        } = this.props;
+
+        return (
+            <>
+                <div className="orders__popup">
+                    <div className="orders__popup-header">
+                        <div className="orders__popup-header_wrapper">
+                            <h2 className="orders__popup-header--title">{title}</h2>
+                            <IconClose className="orders__popup-close_icon" onClick={onCancel} />
+                        </div>
+                    </div>
+                    {need_confirmation ? (
+                        <Formik
```

Comment:
could we define this in a block outside so it doesn't pollute the popup component?

Reply:
the problem is that, the form need to have a button inside it, so both content and footer will be included in the form, ill add todo for this 🙏